### ISSUE TITLE
No release in two years?

### ISSUE THREAD

**Issue by YourGamesBeOver** [mysticatea/cpx#44]
I was wondering why this package hasn't had a new release in a long time? There have been a lot of commits since then, including some dependency cleanup that my current project really needs

**Comment by ghost** [comment]
I agree with @YourGamesBeOver was just coming to ask if we could get a new build.

**Comment by ghost** [comment]
Any update on this @mysticatea

**Comment by kayvanbree** [comment]
Anybody know of any alternatives for this library? I got some issues copying a folder on Jenkins declarative, but if the repo isn't maintained, I won't even bother trying to create an issue.

**Comment by bcomnes** [comment]
I'm maintaining a pristine fork in https://www.npmjs.com/package/cpx2 with hopes to one day reconcile work.
